doc/data-files-description.md

```diff
@@ -9,89 +9,90 @@ This document contains information about all data files associated with this pro
 	+ *Analysis file*: Any file created by a script in `analyses/*`. 
 + **Origin**
 	+ For _PBTA data files_, a link the relevant D3B Center or Kids First workflow (and version where applicable).
-	+ When applicable, a link to the specific *script* that produced (or modified, for *Modified reference file* types) the data.
+	+ When applicable, a link to the specific *module* that produced (or modified, for *Modified reference file* types) the data.
```

Comment:
Since we also have workflows, let's just rephrase this a tad. I don't feel super strongly about this suggestion so whatever you think! 
```suggestion
	+ When applicable, a link to the specific module, workflow, or resource that produced (or modified, for *Modified reference file* types) the data.
```